app/static/js/userProfile.js

```diff
@@ -84,17 +84,25 @@ $(document).ready(function(){
   });
 
   $(".savebtn").click(function () { // Updates the Background check of a volunteer in the database
+    $(this).prop("disabled", true);
     let bgCheckType = $(this).data("id")
     let bgDate = $("#" + bgCheckType + "_date").val()
-    let bgStatus = $("[data-id=" + bgCheckType + "]").val()
-
-    if (bgStatus == '' && bgDate != '') {
-        displayMessage("Status<br>Empty!", "danger")
-        return
+    let bgStatus = $("#" + bgCheckType).val()
+
+    if (bgStatus == '') {
+      $("#" + bgCheckType).addClass("invalid");
+      window.setTimeout(() => $("#" + bgCheckType).removeClass("invalid"), 1000);
+      $("#" + bgCheckType).focus()
+      $(this).prop("disabled", false);
+      return false
     }
-    if (bgStatus != '' && bgDate == '' ) {
-        displayMessage("Date<br>Empty!", "danger")
-        return
+
+    if (bgDate == ''){
+      $("#" + bgCheckType + "_date").addClass("invalid");
+      window.setTimeout(() => $("#" + bgCheckType + "_date").removeClass("invalid"), 1000);
+      $("#" + bgCheckType + "_date").focus()
+      $(this).prop("disabled", false);
+      return false
```

Comment:
In each of these we are selecting the same element multiple times. Save it in a variable at the top so you only have to get it once.